resources/public/api/conf/1.0/application.yaml

```diff
@@ -339,6 +339,49 @@ paths:
       security:
         - userRestricted:
             - trader-goods-profiles
+    delete:
+      tags:
+        - Goods records
+      summary: Withdraw your request for advice from HMRC
+      description: Withdraw your request for advice from HMRC
+      operationId: removeTraderGoodsProfileRecordAdvice
+      parameters:
+        - $ref: '#/components/parameters/Content-Type'
```

Comment:
We shouldn't have content-type if there's no body

Reply:
Fixed